src/main/java/games/strategy/engine/framework/map/download/MapDownloadController.java

```diff
@@ -46,16 +49,15 @@ public boolean checkDownloadedMapsAreLatest() {
 
       SystemPreferences.put(SystemPreferenceKey.TRIPLEA_LAST_CHECK_FOR_MAP_UPDATES, year + ":" + month);
 
-      final List<DownloadFileDescription> downloads =
+      final Collection<DownloadFileDescription> downloadFileDescriptions =
           new DownloadRunnable(mapDownloadProperties.getMapListDownloadSite()).getDownloads();
-
-      final Collection<String> outOfDateMaps = populateOutOfDateMapsListing(downloads);
-      if (!outOfDateMaps.isEmpty()) {
-        final StringBuilder text =
-            new StringBuilder("<html>Some of the maps you have are out of date, and newer versions of those maps exist."
-                + "<br>Would you like to update (re-download) the following maps now?:<br><ul>");
-        for (final String map : outOfDateMaps) {
-          text.append("<li> ").append(map).append("</li>");
+      final Collection<String> outOfDateMapNames = getOutOfDateMapNames(downloadFileDescriptions);
+      if (!outOfDateMapNames.isEmpty()) {
+        final StringBuilder text = new StringBuilder();
+        text.append("<html>Some of the maps you have are out of date, and newer versions of those maps exist.<br>");
+        text.append("Would you like to update (re-download) the following maps now?:<br><ul>");
```

Comment:
Fixing the text to match the behavior,  "would you like to open the download maps window now?" - is downright a bit funny.
That is a problem to solve, solving it now makes sense I think while we are on this topic. 

For consideration, the dialog could have the following three options:
- no/cancel/close window
- open download maps window
- update all

A simplified version of that would be to simply update the current behavior to not only open the window, but also start the download.

Until recently only one map could be downloaded at a time, and it was a UI blocking event. IN that sense it makes sense why this behavior is not a regression, the capability to start downloading multiple maps at once was not there (unless we wanted to be really annoying to the user and block the UI repeatedly)